test/Cobra/DataGen.hs

```diff
@@ -0,0 +1,28 @@
+{-# OPTIONS_GHC -fno-warn-orphans #-}
```

Comment:
Can't we name these with full words, like "DataGenerator"?

Reply:
In quick check the generators are usually suffixed with Gen. Hence the name.